packages/jsii/lib/compiler.ts

```diff
@@ -323,3 +323,18 @@ function nodeJsCompatibleSearchPaths(dir: string): string[] {
 
     return ret;
 }
+
+function parseConfigHostFromCompilerHost(host: ts.CompilerHost): ts.ParseConfigHost {
```

Comment:
Can you add a link to the original implementation? I mean a permalink to the particular location... If it ever needs to be updated, that'll probably help track down where the new version of it lives...